code/modules/busy_space/organizations.dm

```diff
@@ -0,0 +1,461 @@
+//Datums for different companies that can be used by busy_space
+/datum/lore/organization
+	var/name = ""				// Organization's name
+	var/short_name = ""			// Organization's shortname (Nanotrasen for "Nanotrasen Incorporated")
+	var/acronym = ""			// Organization's acronym, e.g. 'NT' for Nanotrasen'.
+	var/desc = ""				// One or two paragraph description of the organization, but only current stuff.  Currently unused.
+	var/history = ""			// Historical description of the organization's origins  Currently unused.
+	var/work = ""				// Short description of their work, eg "an arms manufacturer"
+	var/headquarters = ""		// Location of the organization's HQ.  Currently unused.
+	var/motto = ""				// A motto/jingle/whatever, if they have one.  Currently unused.
+
+	var/list/ship_prefixes = list()	//Some might have more than one! Like Nanotrasen. Value is the mission they perform, e.g. ("ABC" = "mission desc")
+	var/list/ship_names = list(		//Names of spaceships.  This is a mostly generic list that all the other organizations inherit from if they don't have anything better.
+		"Kestrel",
+		"Beacon",
+		"Signal",
+		"Freedom",
+		"Glory",
+		"Axiom",
+		"Eternal",
+		"Icarus",
+		"Harmony",
+		"Light",
+		"Discovery",
+		"Endeavour",
+		"Explorer",
+		"Swift",
+		"Dragonfly",
+		"Ascendant",
+		"Tenacious",
+		"Pioneer",
+		"Hawk",
+		"Haste",
+		"Radiant",
+		"Luminous"
+		)
+	var/list/destination_names = list()	//Names of static holdings that the organization's ships visit regularly.
+	var/autogenerate_destination_names = TRUE
+
+/datum/lore/organization/New()
+	..()
+	if(autogenerate_destination_names) // Lets pad out the destination names.
+		var/i = rand(6, 10)
+		var/list/star_names = list(
+			"Sol", "Alpha Centauri", "Sirius", "Vega", "Regulus", "Vir", "Algol", "Aldebaran",
+			"Delta Doradus", "Menkar", "Geminga", "Elnath", "Gienah", "Mu Leporis", "Nyx", "Tau Ceti",
+			"Wazn", "Alphard", "Phact", "Altair")
+		var/list/destination_types = list("dockyard", "station", "vessel", "waystation", "telecommunications satellite", "spaceport", "distress beacon", "anomaly", "colony", "outpost")
+		while(i)
+			destination_names.Add("a [pick(destination_types)] in [pick(star_names)]")
+			i--
+
+//////////////////////////////////////////////////////////////////////////////////
+
+// TSCs
+/datum/lore/organization/tsc/nanotrasen
+	name = "Nanotrasen Incorporated"
+	short_name = "Nanotrasen"
+	acronym = "NT"
+	desc = "Nanotrasen is one of the foremost research and development companies in SolGov space. \
```

Comment:
desc = "The largest shareholder in the galactic plasma markets, Nanotrasen is a research and mining corporation which specializes in FTL technologies and weapon systems. Frowned upon by most governments due to their shady business tactics and poor ethics record, Nanotrasen is often seen as a necessary evil for maintaining access to the often volatile plasma market. Nanotrasen was originally incorporated on Earth with their headquarters situated on Mars, however they have recently moved most of their operations to the Epsilon Eridani sector."